src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -307,6 +311,88 @@ void run_WithSyncAsArgumentWithAllArg_ShouldExecuteAllSyncers() {
     }
   }
 
+  @Test
+  void run_WithSyncAsArgumentWithCustomersAndShoppingLists_ShouldExecuteGivenSyncers() {
+    // preparation
+    try (final SphereClient targetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
+      try (final SphereClient sourceClient = createClient(CTP_SOURCE_CLIENT_CONFIG)) {
+
+        prepareDataForShoppingListSync(sourceClient);
+        final SyncerFactory syncerFactory =
+            SyncerFactory.of(() -> sourceClient, () -> targetClient, Clock.systemDefaultZone());
+
+        // test
+        CliRunner.of().run(new String[] {"-s", "customers", "shoppingLists"}, syncerFactory);
```

Comment:
wondering more about the case `-s types,  customers, products, shoppingLists" which might make sense for importing shopping lists.

also wondering about a test case that will take all possible values as a "-all" equivalent, which might be good to have and decide for that the SyncFactory will run "-all" 🤔 instead of checking them one by one.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/219/commits/513c4fb8c0b099d7a8416dfa64cd3eb6aeb43492

Reply:
> which might be good to have and decide that the SyncFactory will run "-all" 🤔 instead of checking them one by one.

might be part of the validation logic 🤔 right ?

Reply:
It depends on what is important here. For user to run -all sync, We have already provided an option. If at all he adds all Sync in the arguments by mistake(Which can happen in very rare condition(1/1000)). Then the multiple sync can handle this case. This should not be a problem. Because both approaches are same except the code.
Do you see any other use of adding extra validation for this ?